plugins/module_utils/network/ios/facts/ospfv3/ospfv3.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+The ios ospfv3 fact class
+It is in this file the configuration is collected from the device
+for a given resource, parsed, and the facts tree is populated
+based on the configuration.
+"""
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+from copy import deepcopy
+
+from ansible_collections.ansible.netcommon.plugins.module_utils.network.common import (
+    utils,
+)
+from ansible_collections.cisco.ios.plugins.module_utils.network.ios.argspec.ospfv3.ospfv3 import (
+    Ospfv3Args,
+)
+from ansible_collections.cisco.ios.plugins.module_utils.network.ios.rm_templates.ospfv3 import (
+    Ospfv3Template,
+)
+from ansible_collections.ansible.netcommon.plugins.module_utils.network.common.network_template import (
+    NetworkTemplate,
+)
+from ansible_collections.ansible.netcommon.plugins.module_utils.network.common.utils import (
+    Template,
+    dict_merge,
+)
+
+
+class Ospfv3Facts(object):
+    """ The ios ospfv3 fact class
+    """
+
+    def __init__(self, module, subspec="config", options="options"):
+        self._module = module
+        self.argument_spec = Ospfv3Args.argument_spec
+        spec = deepcopy(self.argument_spec)
```

Comment:
This section of code 41-51 is not needed with the new approach. Please refer https://github.com/ansible-collections/cisco.nxos/blob/main/plugins/module_utils/network/nxos/facts/ospfv3/ospfv3.py#L34. Removing this may help in bumping code cov.

Reply:
updated